lichess.py

```diff
@@ -119,3 +123,11 @@ def get_game_pgn(self, game_id):
         return self.api_get(ENDPOINTS["export"].format(game_id),
                             get_raw_text=True,
                             params={"literate": "true"})
+
+    def get_online_bots(self):
```

Comment:
Is it possible for the engine controlled by lichess-bot to be returned from this list? In other words, could the local engine end up challenging itself?

Reply:
It was possible to happen. Now it is fixed.